systests/jaxws/src/test/java/org/apache/cxf/systest/jaxws/websocket/ClientServerWebSocketTest.java

```diff
@@ -54,7 +54,9 @@
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
-
+//TODO: Enable this test after jakarta websocket is ready
+@org.junit.Ignore("It requires jakarta version:The CXF websocket transport needs either the "
```

Comment:
I am wondering how we consolidate all ignored tests cases and commented out code eventually? May be we could let them sit there but fail ?

Reply:
I'll move these ignore annotation, and still leave there the comment to indicate these jakarta dependency missing.